astroquery/mast/core.py

```diff
@@ -82,31 +81,6 @@ def logout(self):
         self._auth_obj.logout()
         self._authenticated = False
 
-    def enable_cloud_dataset(self, provider="AWS", profile=None, verbose=True):
```

Comment:
I would say these should be deprecated rather than starting to break user code without notice; while these have no effect, the code do run with non-cloud returns, right?

The deprecations would work nicely if you keep overriding the methods in the classes that these work and not get the deprecation.

What do you think @pllim?

Reply:
Yes, I think these should be deprecated first. It sounds like people might be using these in the wild whether intentionally or not.

Reply:
This makes sense! The latest version deprecates the functions in the base class.

Reply:
Thanks! Hmm you could also use `@deprecated` decorator from `astropy.utils` but I'll leave that to astroquery maintainers to decide.

Reply:
What's your preference @bsipocz? It looks like there are instances of both methods in the repo.

Reply:
Made that change! I followed the convention of the other deprecated functions and left the original docstrings in.